vello/src/scene.rs

```diff
@@ -511,7 +511,8 @@ impl<'a> DrawGlyphs<'a> {
 
                             if !mask.is_packed {
                                 // TODO: Error once?
-                                log::warn!("Unpacked mask data not handled");
+                                log::warn!("Unpacked mask data in font not yet supported");
+                                // TODO: How do we get the font name here?
```

Comment:
If we were using `tracing`, we'd be able to have some context data around that made this more clear, perhaps? (Clearly not a thing for this PR to solve.)

Reply:
I suspect that getting that information "just-in-case" all the time would be too expensive. This is a potentially hot-spot in the API.

Reply:
Ideally, it would be somewhere further up the call stack, although for fonts, maybe not quite far enough.